judges/cuda/test.yaml

```diff
@@ -0,0 +1,30 @@
+memory:       512MB
+memory_gpu:   256MB
+cuda_architecture: 'sm_52'
+kolejka:
+    image       : 'kolejka/satori:extended'
+    exclusive   : false
+    requires    : [ 'gpu:nvidia' ]
+    collect     : [ 'log.zip' ]
+    limits:
+        time        : '20s'
+        memory      : '4G'
+        swap        : 0
+        cpus        : 4
+        network     : false
+        pids        : 256
+        storage     : '2G'
+        workspace   : '2G'
+        gpus        : 1
+        memory_gpu  : '256MB'
+    satori:
+        result:
+            execute_time_real : '/io/executor/run/real_time'
+            execute_time_cpu  : '/io/executor/run/cpu_time'
+            execute_memory    : '/io/executor/run/memory'
+            execute_time_gpu  : '/io/executor/run/gpu_time'
```

Comment:
remove?

Reply:
I'll change it as staded in another comment -> try to get metric from profiler